capstone/src/main/webapp/image_analyzer.html

```diff
@@ -19,30 +19,36 @@
       <header class="masthead mb-auto">
         <div class="inner">
           <div class="text-center">
-            <img src="assets/perspective_horizontal_color.png" class="rounded">
+            <a href="http://perspectiveapi.com/#/home">
+              <img src="assets/perspective_horizontal_color.png" class="rounded">
+            </a>
           </div>
           <h3 class="masthead-brand">Perspective Hub</h3>
           <nav class="nav nav-masthead justify-content-center">
             <a class="nav-link" href="index.html">Home</a>
             <a class="nav-link" href="text_analyzer.html">Text Analyzer</a>
             <a class="nav-link active" href="image_analyzer.html">Image Analyzer</a>
             <a class="nav-link" href="document_analyzer.html">Document Analyzer</a>
-            <a class="nav-link" href="https://support.perspectiveapi.com/s/" target="_blank">Support</a>
-            <a class="nav-link" href="https://medium.com/the-false-positive" target="_blank">Blog</a>
             <a class="nav-link" href="about.html">About</a>
           </nav>
         </div>
       </header>
-      <h2>Attach an image and select the Perspective API attribute(s) you would like to be shown</h2>
+      <main role="main">
+        <section class="jumbotron text-center">
+          <div class="container">
+            <h1>Image Analysis</h1>
+            <p class="lead text-muted">Upload an image and select the Perspective API attribute(s) you would like to be shown</p>
+            <small id="disclaimer" class="form-text text-muted">Supported Formats: .JPEG, .JPG, .PNG</small>
```

Comment:
Depending on API/implementation details, this might have to change later on. For now, this is good!

Reply:
Noted! I'll keep that in mind